enterprise/cypher/physical-planning/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/SlottedRewriter.scala

```diff
@@ -222,6 +223,12 @@ class SlottedRewriter(tokenContext: TokenContext) {
       case e@IsNull(Property(Variable(key), PropertyKeyName(propKey))) =>
         Not(checkIfPropertyExists(slotConfiguration, key, propKey))(e.position)
 
+      case _: ReduceExpression =>
```

Comment:
think we should merge these last 4 cases into a single one:
```
case _: ReduceExpression | _: DesugaredMapProjection | _: ShortestPathExpression | _: PatternExpresion =>
         throw new CantCompileQueryException(s"This expression is not yet supported in the slotted runtime")
```

Reply:
Maybe, although you would get a less useful stack trace when you are looking at why something was not supported. But we have already removed some of these in the next PR we are currently working on. This is just a quick fix for the docs build.